cdk8s/lib/index.js

```diff
@@ -0,0 +1,34 @@
+const cdk8s = require('cdk8s');
+const { core } = require('@winglang/sdk');;
+
+exports.Platform = class {
+  target = "cdk8s";
+
+  newApp(props) {
+    const app = new cdk8s.App({ outdir: props.outdir });
+    const chart = new cdk8s.Chart(app, "Chart");
+
+    class App extends core.App {
+      _target = "cdk8s";
+      outdir = props.outdir;
+
+      constructor() {
+        super(chart, "App", props);
+
+        const root = this.node.root;
+        root.new = (fqn, ctor, scope, id, ...args) => this.new(fqn, ctor, scope, id, ...args);
+        root.newAbstract = (fqn, scope, id, ...args) => this.newAbstract(fqn, scope, id, ...args);
+        root.typeForFqn = fqn => this.typeForFqn(fqn);
```

Comment:
Interesting, I wonder what happens if you try defining a `test` or something like that